python/ola/ClientWrapperTest.py

```diff
@@ -21,7 +21,11 @@
 import datetime
 import socket
 # import timeout_decorator
-import unittest
+try:
+  # Python 2.6 needs backport
+  import unittest2 as unittest
+except ImportError:
+  import unittest
```

Comment:
@brucelowekamp may have an opinion between try/execept and sys.version based checks. They may have an opinion about doing this at all too! :smile: 

I also wonder if moving it into ola.TestUtils might work too, to keep the logic once in one place, or if that might upset flake8/pychecker.